README.md

```diff
@@ -13,15 +22,71 @@ notebooks in this repository:
 1. Install [Anaconda](https://docs.anaconda.com/anaconda/install/)
    or [Miniconda](https://docs.conda.io/en/latest/miniconda.html)
 1. Navigate to your local copy of this repository.
-1. Run the script `env.sh` to create an Anaconda environment in the directory
-   `./env`:
+1. Run the script `env.sh` to create an Anaconda environment in the directory `./env`:
    ```console
    $ bash env.sh
    ```
    Note: This script takes a while to run.
 1. Activate the new environment and start JupyterLab:
    ```console
    $ conda activate ./env
-   $ jupyter lab
+   $ jupyter lab --debug
    ```
 
+#### Configuring a local Kubeflow Pipeline runtime
+
+[Elyra's Notebook pipeline visual editor](https://elyra.readthedocs.io/en/latest/getting_started/overview.html#notebook-pipelines-visual-editor)
+currently supports running these pipelines in a Kubeflow Pipeline runtime,
+and, if required, these are [the steps to install a local deployment of KFP](https://elyra.readthedocs.io/en/latest/developer_guide/deploying-kubeflow-locally-for-dev.html).
+
```

Comment:
Since you have instructions for installing Kubeflow on Docker Desktop, I recommend that you make the primary path through the remainder of the documentation assume that users are running all Kubeflow Pipelines jobs on their local machine. Configuration will be much simpler then. You can probably even automate most of the setup.